subsys/bluetooth/host/hci_core.c

```diff
@@ -4083,6 +4083,21 @@ static void hci_cmd_status(struct net_buf *buf)
 }
 
 #if defined(CONFIG_BT_OBSERVER)
+static inline bool is_adv_using_rand_addr(void)
```

Comment:
I'd drop the `inline` here and let the compiler take care of it (it will most certainly inline it anyway). IMO it's only in cases where you do careful profiling after the fact and discover you get a significant benefit out of explicit inlining that it's justified to do so.

Reply:
Agree, removed it.